src/entities/SecurityToken/Issuance/Offerings.ts

```diff
@@ -136,24 +141,14 @@ export class Offerings extends SubModule {
   /**
    * Launch a Tiered STO
    *
-   * @param startDate date when the STO should start
-   * @param endDate date when the STO should end
-   * @param tiers tier information
-   * @param tiers[].tokensOnSale amount of tokens to be sold on that tier
-   * @param tiers[].price price of each token on that tier
-   * @param tiers[].tokensWithDiscount amount of tokens to be sold on that tier at a discount if paid in POLY (must be less than tokensOnSale, defaults to 0)
-   * @param tiers[].discountedPrice price of discounted tokens on that tier (defaults to 0)
-   * @param nonAccreditedInvestmentLimit maximum investment for non-accredited investors
-   * @param minimumInvestment minimum investment amount
-   * @param currencies array of currencies in which the funds will be raised (ETH, POLY, StableCoin)
-   * @param raisedFundsWallet wallet address that will receive the funds that are being raised
-   * @param unsoldTokensWallet wallet address that will receive unsold tokens when the end date is reached
-   * @param stableCoinAddresses array of stable coins that the offering supports
-   * @param customCurrency custom currency data. Allows the STO to raise funds pegged to a different currency. Optional, defaults to USD
-   * @param customCurrency.currencySymbol symbol of the custom currency (USD, CAD, EUR, etc. Default is USD)
-   * @param customCurrency.ethOracleAddress address of the oracle that states the price of ETH in the custom currency. Only required if raising funds in ETH
-   * @param customCurrency.polyOracleAddress address of the oracle that states the price of POLY in the custom currency. Only required if raising funds in POLY
-   * @param allowPreIssuance whether to have all tokens issued on STO start. Default behavior is to issue on purchase
+   * @param args.nonAccreditedInvestmentLimit - maximum investment for non-accredited investors
+   * @param args.minimumInvestment - minimum investment amount
+   * @param args.currencies - array of currencies in which the funds will be raised (ETH, POLY, StableCoin)
+   * @param args.raisedFundsWallet - wallet address that will receive the funds that are being raised
+   * @param args.unsoldTokensWallet - wallet address that will receive unsold tokens when the end date is reached
+   * @param args.stableCoinAddresses - array of stable coins that the offering supports
+   * @param args.customCurrency - it can be optional
```

Comment:
Why delete the rest of the description? The previous one was correct. Anything related to the contents of the object should be covered in the docs for the `CustomCurrency` interface